lint/flagparser/flag_parser.go

```diff
@@ -0,0 +1,84 @@
+package flagparser
```

Comment:
I think goling is yelling that there is no docs for exported declarations 😉

Reply:
Done. Why it is not in travis build?

Also as I don't see there gofmt checks, golint checks, etc.